### ISSUE TITLE
Warning: Your Android version does not allow syncing to external storage devices

### ISSUE THREAD

**Issue by Core-Xii** [syncthing/syncthing-android#1850]

**Comment by ddolcimascolo** [comment]
Happened to me today. I'm using "Share with" Syncthing daily and it worked fine before.

Any news on this?

**Comment by ddolcimascolo** [comment]
> Happened to me today. I'm using "Share with" Syncthing daily and it worked fine before.
> 
> Any news on this?

Sorry, I do have the message in the app, but my issue is something else. Maybe I always had that message and didn't pay attention...

**Comment by steel-steven-87** [comment]
The recent update broke something. I can't save any files using "Share with" anymore and I get the same message. I also noticed that the browse button is not fully visible:
![image](https://github.com/syncthing/syncthing-android/assets/89798696/4f1c94d6-3d5e-42d2-9135-3d19ca13ec16)

**Comment by ddolcimascolo** [comment]
@steel-steven-87 Yeah, there are buttons at the bottom, but they're not visible. See my other issue #1918

**Comment by steel-steven-87** [comment]
@ddolcimascolo Haha, that's what I call a clean design! 😄

Well, I'm glad there's a workaround, so I can back up the data.

If anyone needs to save, just click around on the bottom right side of the screen to find the invisible save button.

**Comment by r4taw2mg** [comment]
I have this too, on two devices: Android 8 and Android 10. Only ever updated syncthing.

**Comment by cizmazia** [comment]
I have started experiencing this on Android 12 when sharing a file from another app via the option "Save to Syncthing" . It worked fine previously.

**Comment by lmiol** [comment]
same!

**Comment by r4taw2mg** [comment]
> I have this too, on two devices: Android 8 and Android 10. Only ever updated syncthing.

Still not working in 1.23.6 . I'd really appreciate somebody looking into this.

**Comment by dilectia** [comment]
Same problem here using Grapheneos with Android 13 in a Pixel 6a

**Comment by bt90** [comment]
Seems like the folder picker was not adapted work with anything past Android KitKat:

https://github.com/syncthing/syncthing-android/blob/5847e74905b3fe7ddeb92c6ece85e73e48162d80/app/src/main/java/com/nutomic/syncthingandroid/activities/FolderPickerActivity.java#L107

I think @Catfriend1 fixed this one in the fork: https://github.com/Catfriend1/syncthing-android/issues/690

**Comment by LLCodeWiz** [comment]
bump

**Comment by combs** [comment]
This just happened to me on a Samsung A54 5G running Android 14. It began showing this warning and my folder was unexpectedly changed to "Send Only." I was able to fix it by navigating through the Web GUI to the folder's Advanced tab and choosing "Send/Receive" again. It worked fine after that. Unsure if this will help you, but just in case...

**Comment by bw35** [comment]
Changing the path of the shared folder from "~/Default Folder" to "storage/emulated/0/Default Folder" did the trick for me. Seems that the tilde is causing the problem. The folder needs to be removed and added again to change the path.

**Comment by fauxmight** [comment]
@bw35 Android 14 here. This is a functioning work-around for the issue in my instance. Thank you.
